### PR TITLE
GTFS change highlights backend

### PR DESCRIPTION
**Pull request by moradology**

### CODE REVIEW

File: deployment/gtfs_delta.sql [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+-- Name: gtfs_delta; Type: TABLE; Schema: public; Owner: transit_indicators; Tablespace:
+--
+
+CREATE TABLE public.gtfs_delta (
+  id text PRIMARY KEY,
+  deltaType integer NOT NULL,
+  geom geometry(LineString,4326) NOT NULL,
+);
+
+ALTER TABLE public.station_csv OWNER TO transit_indicators;
```
Comment (ddohler) [inline deployment/gtfs_delta.sql]: This should probably be broken out into a separate commit, it looks like it's a bug fix.

Comment (moradology) [inline deployment/gtfs_delta.sql]: Actually, I think that's just a typo! Good catch

File: scala/opentransit/src/main/scala/com/azavea/opentransit/database/GTFSDeltaTable.scala [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+import geotrellis.vector._
+
+import scala.slick.driver.{JdbcDriver, JdbcProfile, PostgresDriver}
+import scala.slick.jdbc.JdbcBackend.DatabaseDef
+
+
+// The performance characteristics of postgres enums should be the same as integers.
+// Instead of introducing a dependency for PG enums (something we should do from the start),
+// I'll use typesafety and the `DeltaType` sum type to achieve parity.
+sealed trait DeltaType { val intRep: Int }
```
Comment (notthatbreezy) [inline scala/opentransit/src/main/scala/com/azavea/opentransit/database/GTFSDeltaTable.scala]: Nice use of sealed trait

**Comment by notthatbreezy** [comment]
+1 - was able to add changes and delete routes, verified changes in new delta tables.
